src/01-practices/codyMan0/2주차/cart/Cart.tsx

```diff
@@ -0,0 +1,20 @@
+// import styles from './Cart.module.css';
+// import styles from './Cart.module.css';
+import { CartItem } from '../cartItem/CartItem';
+import { RestaurantInfo } from '../type';
+
+interface CartProps {
+  restaurantList: RestaurantInfo;
```

Comment:
`restaurantList`가 목록 형태가 아니라서 이름을 바꾸는 게 좋을 것 같네요
```suggestion
  restaurant: RestaurantInfo;
```

Reply:
넵!